frame/collective/src/lib.rs

```diff
@@ -505,6 +518,7 @@ decl_module! {
 			#[compact] index: ProposalIndex,
 			approve: bool,
 		) -> DispatchResultWithPostInfo {
+			let mut is_account_voting_first_time: bool = false;
```

Comment:
```suggestion
```

Reply:
Dont use mutable variables like this if you dont need to

Reply:
Thanks Shawn will correct my bad coding style ...